### PR TITLE
Sorted nearest

### PR DESCRIPTION
**Pull request by endrebak**
Checklist

- [*] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [*] Source is from official source
- [*] Package does not vendor other packages
- [*] Build number is 0
- [*] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/sorted_nearest/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "dc8c765ee7c56738fa5be16ec453e18978f1ecc7675832ab4c0cdf3a50115727" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.{{ file_ext }}'
```
Comment (chrisburr) [inline recipes/sorted_nearest/meta.yaml]:
```suggestion
```

File: recipes/sorted_nearest/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
```
Comment (chrisburr) [inline recipes/sorted_nearest/meta.yaml]:
```suggestion
  sha256: dc8c765ee7c56738fa5be16ec453e18978f1ecc7675832ab4c0cdf3a50115727
```

File: recipes/sorted_nearest/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,49 @@
+{% set name = "sorted_nearest" %}
+{% set version = "0.0.17" %}
+{% set file_ext = "tar.gz" %}
```
Comment (chrisburr) [inline recipes/sorted_nearest/meta.yaml]: Can you remove these three lines (`file_ext`, `hash_type` and `hash_value`)

File: recipes/sorted_nearest/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+{% set hash_type = "sha256" %}
+{% set hash_value = "dc8c765ee7c56738fa5be16ec453e18978f1ecc7675832ab4c0cdf3a50115727" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.{{ file_ext }}'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
```
Comment (chrisburr) [inline recipes/sorted_nearest/meta.yaml]:
```suggestion
  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
```

File: recipes/sorted_nearest/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
```
Comment (chrisburr) [inline recipes/sorted_nearest/meta.yaml]: If you don't want to build this for Windows you can do:
```suggestion
  skip: true  # [win]
```

File: recipes/sorted_nearest/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+    - {{ compiler('c') }}
+  host:
+    - python
+    - setuptools
+    - cython
+    - numpy
+  run:
+    - python
+    - cython
+    - numpy
```
Comment (chrisburr) [inline recipes/sorted_nearest/meta.yaml]: http://conda-forge.org/docs/maintainer/knowledge_base.html?highlight=numpy#building-against-numpy
```suggestion
    - {{ pin_compatible('numpy') }}
```

File: recipes/sorted_nearest/meta.yaml [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+  imports:
+    - sorted_nearest
+
+about:
+  home: https://github.com/endrebak/sorted_nearest
+  license: New BSD
+  license_family: BSD
+  license_file: LICENSE.txt
+  summary: Find nearest interval.
+  description: ''
```
Comment (chrisburr) [inline recipes/sorted_nearest/meta.yaml]: Can you remove the empty fields

File: recipes/sorted_nearest/meta.yaml [hunk 8]
```diff
@@ -0,0 +1,49 @@
...
+    - cython
+    - numpy
+
+test:
+  imports:
+    - sorted_nearest
+
+about:
+  home: https://github.com/endrebak/sorted_nearest
+  license: New BSD
```
Comment (chrisburr) [inline recipes/sorted_nearest/meta.yaml]:
```suggestion
  license: BSD-3-Clause
```

**Comment by endrebak** [comment]
Thanks. This I'll incorporate your suggested changes into my recipes from now on :)

**Comment by chrisburr** [comment]
Did you mean to close this?

**Comment by endrebak** [comment]
Yes, I added sorted_nearest to bioconda since I felt it was more appropriate there. But I am a conda-forge contrib and will use your tips in the future. Thanks :)

**Pull request closed without merging**